core/services/pipeline/task.lessthan.go

```diff
@@ -0,0 +1,50 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+)
+
+// Return types:
+//
+//	bool
+type LessThanTask struct {
+	BaseTask `mapstructure:",squash"`
+	Input    string `json:"input"`
+	Limit    string `json:"limit"`
```

Comment:
To me these parameter names are confusing... maybe we can name them `lhs`, `rhs`? or just left/right?

Reply:
done